### ISSUE TITLE
[go-borges] Fix tests in integration

### ISSUE THREAD

**Issue by jfontan** [src-d/gitbase#857]
* ~~Create branch in gitbase repository with the initial integration from https://github.com/jfontan/gitbase/tree/go-borges-go-rebased~~ is `go-borges-integration`
* Tests from `cmd/gitbase/command/server_test.go` are still not modified to use go-borges. I'm not sure these are needed anymore.
* Tests that use `closedSiva` can not work with go-borges (`testTableIndexIterClosed` and `testTableIterClosed`). They may be moved to go-borges.
* `TestRefsIterSiva` fails and I'm not sure what's needed to make it work.

```
--- FAIL: TestRefsIterSiva (0.00s)
    --- FAIL: TestRefsIterSiva/all_refs (0.00s)
        require.go:97: 
            	Error Trace:	squash_iterator_test.go:989
            	Error:      	element [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] appears more times in [[/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3]] than in [[015da2f4-6d89-7ec8-5ac9-a38329ea875b HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [015da2f4-6d89-7ec8-5ac9-a38329ea875b refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3]]
            	Test:       	TestRefsIterSiva/all_refs
    --- FAIL: TestRefsIterSiva/repo_refs (0.00s)
        require.go:97: 
            	Error Trace:	squash_iterator_test.go:989
            	Error:      	element [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] appears more times in [[/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3]] than in [[015da2f4-6d89-7ec8-5ac9-a38329ea875b HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [015da2f4-6d89-7ec8-5ac9-a38329ea875b refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3]]
            	Test:       	TestRefsIterSiva/repo_refs
    --- FAIL: TestRefsIterSiva/remote_refs (0.00s)
        require.go:97: 
            	Error Trace:	squash_iterator_test.go:1013
            	Error:      	element [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] appears more times in [[/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3] [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3] [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3]] than in [[015da2f4-6d89-7ec8-5ac9-a38329ea875b HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [015da2f4-6d89-7ec8-5ac9-a38329ea875b refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3] [015da2f4-6d89-7ec8-5ac9-a38329ea875b HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [015da2f4-6d89-7ec8-5ac9-a38329ea875b refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3] [015da2f4-6d89-7ec8-5ac9-a38329ea875b HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [015da2f4-6d89-7ec8-5ac9-a38329ea875b refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3]]
            	Test:       	TestRefsIterSiva/remote_refs
```

**Comment by jfontan** [comment]
Now there's a branch in gitbase `go-borges-integration`.

**Comment by kuba--** [comment]
It happens because as far as I know we changed `repository_id` to full path.

**Comment by jfontan** [comment]
Fixed in #888
